src/app/(proper_react)/redesign/(authenticated)/user/dashboard/page.tsx

```diff
@@ -72,6 +74,14 @@ export default async function DashboardPage() {
 
   await refreshStoredScanResults(profileId);
 
+  // If the current user is a subscriber and their OneRep profile is not
+  // activated: Most likely we were not able or failed to kick-off the
```

Comment:
How are we detecting if the user is activated or not now? It should be in the profile so we could check that with an API call, but how do we know if `opt-out` didn't work? The lack of `initial` scan would work I think... I don't see where the code is doing any checks like this though?

Reply:
I’ve added a check for the profile status here: https://github.com/mozilla/blurts-server/pull/3705/files#diff-88d1915de743c76564f89f25f640df3935f835fc8db3609ae1763c893a1dfdf1R223-R224.

Reply:
Also added the check to only `opt-out` a profile when there is no existing `initial` scan in [24f3b68](https://github.com/mozilla/blurts-server/pull/3705/commits/24f3b6848a0d03540baa79f72b09bb7187056cbc).